Source/KerbalCombatSystems/RocketTargeting.cs

```diff
@@ -84,7 +86,7 @@ public override Vector3 Aim()
             Vector3 targetRadius = Vector3.ProjectOnPlane(Vector3.up, targetVector.normalized).normalized * (controller.targetSize / 2) * accuracyTolerance;
             float aimTolerance = Vector3.Angle(targetVector, targetVector + targetRadius);
 
-            bool onTarget = Vector3.Angle(leadVector.normalized, decoupler.transform.up) < aimTolerance;
+            bool onTarget = Vector3.Angle(leadVector.normalized, aimVector) < aimTolerance;
```

Comment:
```suggestion
            bool onTarget = Vector3.Angle(leadVector.normalized, decoupler.transform.up) < aimTolerance;
```
Because the ship isn't currently aiming with aimVector, onTarget will never be true unless the difference between aimVector and the current reference transform is less than aimTolerance. This should probably be reverted until your re-organisation of rocket targeting.